packages/lumx-react/src/components/notification-block/NotificationBlock.stories.tsx

```diff
@@ -6,25 +6,26 @@ import { NotificationBlock } from './NotificationBlock';
 
 export default { title: 'LumX components/NotificationBlock' };
 
-export const icon = () => {
+export const icon = ({ theme }: any) => {
```

Comment:
Is it required in the DS to have TSX for stories? 
You can use JSX I guess

Reply:
I think it's useful to catch minor typing error on component prop values. Plus the only inconvenience is this `any` input for the stories